src/js/main.js

```diff
@@ -193,607 +198,608 @@
+
+            function addBackgroundToScene(scene) {
+                var geometry = new THREE.PlaneBufferGeometry(35, 35);
+                var ground = new THREE.Mesh(geometry, constant.groundMaterial);
+                ground.position.set(0, 0, -0.01);
+                //ground.rotation.x = - Math.PI / 2;
+                //scene.add(ground);
+                scene.fog = new THREE.Fog(0xffffff, 0, 1000);
+                scene.background = new THREE.Color(0xcce0ff);
             }
-        }
 
-        function addBackgroundToScene(scene) {
-            var geometry = new THREE.PlaneBufferGeometry(35, 35);
-            var ground = new THREE.Mesh(geometry, constant.groundMaterial);
-            ground.position.set(0, 0, -0.01);
-            //ground.rotation.x = - Math.PI / 2;
-            //scene.add(ground);
-            scene.fog = new THREE.Fog(0xffffff, 0, 1000);
-            scene.background = new THREE.Color(0xcce0ff);
-        }
+            function addFloorplanToScene(scene, floorPlan) {
+                scene.add(floorPlan)
+            }
 
-        function addFloorplanToScene(scene, floorPlan) {
-            scene.add(floorPlan)
-        }
+            function addWallsToScene(scene, walls) {
+                for (const wall of walls) {
+                    scene.add(wall)
+                }
+            }
 
-        function addWallsToScene(scene, walls) {
-            for (const wall of walls) {
-                scene.add(wall)
+            function addObjectsToScene(scene, objects) {
+                for (const object of objects) {
+                    scene.add(object)
+                }
             }
-        }
 
-        function addObjectsToScene(scene, objects) {
-            for (const object of objects) {
-                scene.add(object)
+            // Text
+            function createText(text) {
+                const textGeometry = createTextGeometry(text, roomModel.font);
+                const textMesh = new THREE.Mesh(textGeometry, constant.textMaterial)
+                return textMesh;
             }
-        }
 
-        // Text
-        function createText(text) {
-            const textGeometry = createTextGeometry(text, roomModel.font);
-            const textMesh = new THREE.Mesh(textGeometry, constant.textMaterial)
-            return textMesh;
-        }
+            function createTextGeometry(text, font) {
+                var textGeometry = new THREE.TextGeometry(text, {
+                    font: font,
+                    size: constant.textSize,
+                    height: 0
+                });
+                return textGeometry
+            }
+        };
 
-        function createTextGeometry(text, font) {
-            var textGeometry = new THREE.TextGeometry(text, {
-                font: font,
-                size: constant.textSize,
-                height: 0
-            });
-            return textGeometry
-        }
+        // Preprocessing - Live JSON
 
```

Comment:
all functions were moved into `init`, so the only thing exposed (to call) is init